default/urls.py

```diff
@@ -7,6 +7,9 @@
     path('i18n/', include('django.conf.urls.i18n')),
     path('upload/', views.uploadfile, name='upload'),
     path('delete/<uuid:id>', views.deletefile, name='delete_file'),
+    path('result/<str:folder>/<uuid:id>/drive/', views.retrieve_result_drive, name='retrieve_result_drive'),
+    path('result/<str:folder>/<uuid:id>/<str:format>/drive/', views.retrieve_result_drive,
```

Comment:
Instead of having new URLs and methods, you can receive a GET parameter in the usual download URL, check Django's documentation.